app/javascript/components/Greetings.js

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchRandomGreeting  } from '../store/greetings/greetingSlice';
+import '../../assets/stylesheets/application.css';
+
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you if you remove this code line from your code base as it is the one generating the error mentioned by the previous reviewer. Your style is working even without this.
![image](https://github.com/ClaudiaRojasSoto/Hello_Rails_React/assets/83042742/b74b3673-867a-4b33-a955-c129a51a2e37)

Reply:
Thank you again @tresorsawasawa!!! I will do it!